snippets/randomIntWithLengthInRange.md

```diff
@@ -0,0 +1,21 @@
+### randomIntWithLengthInRange
```

Comment:
I suggest we name this `randomIntArrayInRange`, which is not too verbose and should be easy to find.